homework_2.js

```diff
@@ -0,0 +1,68 @@
+// 1
+// Функция принимает массив чисел. Возвращает массив уникальных чисел
+// fn([1,1,1,2,2,2,3,4,5,5,6]) // [1,2,3,4,5,6]
+
+const uniqueList = (list) => {
```

Comment:
in naming of functions should be kind of action what the function do. 
HINTS: `createUniqueList` or 'getUniqueNumbers'